### PR TITLE
add 10 electrons gun gen fragment

### PR DESCRIPTION
**Pull request by mtosi**
as requested by EGM for both offline and HLT reconstruction/ID/etc validation
this is the gen fragment for generating events w/ 10 electrons w/
- energy [0;200] GeV
- |eta| < 3.
this gen fragment is a copy-n-paste of the [TenMu gun](https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/Configuration/Generator/python/TenMuE_0_200_pythia8_cfi.py)

I simply changed -13 into -11
and increased the eta acceptance

this will be used for standard relval samples for release validation
and it will help the development and tuning of the EGM reco/ID/etc

@paramatti @fcouderc @gpasztor

### CODE REVIEW

**Comment by govoni** [comment]
Ciao Mia,
I have a comment ad a question:
- May I suggest to change the EDFilter name, so that it's different from the muons one?
- Was the fragment tested locally?
Cheers

**Comment by rafaellopesdesa** [comment]
@govoni  Hi Pietro, the generator fragment runs ok. This is the output of running 10 events using the same configuration as the TenMuE wf:

https://rcoelhol.web.cern.ch/rcoelhol/TenEE/?match=Electron

It looks healthy for EGM. About filter names and other technical details, that is outside my expertise.

**Comment by mtosi** [comment]
thanks @govoni and @rafaellopesdesa !
I've just fixed the configuration by adding the module name
(sorry for this issue)

I need to backport this PR to 80x and 81x
because the EGM group requested this sample for online and offline development for 2017 data taking

File: Configuration/Generator/python/TenE_E_0_200_pythia8_cfi.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import FWCore.ParameterSet.Config as cms
+
+# Modified from Configuration/Generator/python/TenMuE_0_200_pythia8_cfi.py
+generator = cms.EDFilter("Pythia8EGun",
```
Comment (govoni) [inline Configuration/Generator/python/TenE_E_0_200_pythia8_cfi.py]: Ciao Mia, isn't "Pythia8EGun" the same name used for the ten muon gun EDFilter? Wouldn't it be better to change the name here?

**Comment by govoni** [comment]
@rafaellopesdesa thanks a lot for the plots!
@mtosi I put a comment directly in the code, about the EDFilter name, would you please check it?

File: Configuration/Generator/python/TenE_E_0_200_pythia8_cfi.py (hunk 1 above)
Comment (mtosi) [inline Configuration/Generator/python/TenE_E_0_200_pythia8_cfi.py]: I'm confused by your comment, that name is not the name of the module
(and actually, I checked and I saw all fragments are using "generator" as name and I find it as expected, actually)
but it is the name of the CMSSW template which defines the type of EDFilter we want to run ...

Comment (govoni) [inline Configuration/Generator/python/TenE_E_0_200_pythia8_cfi.py]: I don't know why you need the string in general, but I think that you can choose it. See for example here:
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideGenFilters

Anyhow, as one either produces ten muons or ten electrons, I don't see a big overlap problem here.

**Comment by govoni** [comment]
+1

**Comment by davidlange6** [comment]
assign dmv

(just to confirm that @fabozzi is also in agreement)

**Comment by davidlange6** [comment]
assign pdmv

**Comment by fabozzi** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**